engine-dmn/feel-scala/pom.xml

```diff
@@ -64,4 +68,30 @@
 
   </dependencies>
 
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.felix</groupId>
+        <artifactId>maven-bundle-plugin</artifactId>
+        <configuration>
+          <instructions>
+            <Import-Package>
+              !scala.*,
```

Comment:
As mentioned in `engine-dmn/engine/pom.xml`, I am not entirely sure yet if this is necessary as I cannot see any entries of `scala.*` in the import directive but rather only `camundajar.impl.scala.*` entries in `uses` attributes in the export directive.

Reply:
Yes, you're right, excluding scala package is not correct. Unfortunately package camundajar.impl.scala.* from feel-engine-*.jar is not an OSGi bundle. So we should convert feel-engine*.jar to OSGi bundle as well.

Reply:
If you like, you can open another ticket in our tracker and provide a PR to add such a capability in the respective project. I would consider this out of scope of this ticket here.